2-Browsers/Week1/homework/ex1-bookList/style.css

```
@@ -1 +1,20 @@
 /* Write your style here */
+ul {
+  display: flex;
+}
+li {
+  width: 400px;
```

Maybe we can change the width here to something:
```suggestion
  width: calc(25% - 51px);
```